[2020-07-13 23:43:04,363] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: news_fetch.push_s3 2020-07-11T01:53:00+00:00 [queued]>
[2020-07-13 23:43:04,376] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: news_fetch.push_s3 2020-07-11T01:53:00+00:00 [queued]>
[2020-07-13 23:43:04,376] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-07-13 23:43:04,376] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-07-13 23:43:04,376] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-07-13 23:43:04,383] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): push_s3> on 2020-07-11T01:53:00+00:00
[2020-07-13 23:43:04,385] {standard_task_runner.py:53} INFO - Started process 30752 to run task
[2020-07-13 23:43:04,420] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: news_fetch.push_s3 2020-07-11T01:53:00+00:00 [running]> tony-ThinkPad-X1-Carbon-7th
[2020-07-13 23:43:08,210] {python_operator.py:114} INFO - Done. Returned value was: True
[2020-07-13 23:43:08,221] {taskinstance.py:1065} INFO - Marking task as SUCCESS.dag_id=news_fetch, task_id=push_s3, execution_date=20200711T015300, start_date=20200713T154304, end_date=20200713T154308
[2020-07-13 23:43:14,369] {logging_mixin.py:112} INFO - [2020-07-13 23:43:14,368] {local_task_job.py:103} INFO - Task exited with return code 0
